vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -283,6 +284,6 @@ export WORKERS_KUBEADM_EXTRA_CONFIG="
           permissions: '0644'
           content: |
             {
-              \"insecure-registries\" : [\"192.168.111.1:5000\"]
+              \"insecure-registries\" : ["{{ REGISTRY }}"]
```

Comment:
Not sure, but maybe this might be the reason why CI is failing, WDYT?:
```suggestion
              \"insecure-registries\" : [\"{{ REGISTRY\ }}"]
```

Reply:
Hum, it passed my local tests though.

Reply:
@furkatgofurov7 Seems you were right. Added backslash now, let's see how it goes in the CI this time.